ros2/src/airsim_ros_pkgs/include/math_common.h

```diff
@@ -42,4 +42,13 @@ inline T angular_dist(T from, T to)
         d += 2. * M_PI;
     return d;
 }
+
+template <typename T>
+inline T constrain_euler_angle_to_360(const T angle)
+{
+    T angle_constrained = angle;
+    angle_constrained -= 360. * std::floor(angle_constrained * (1. / 360.));

```

Comment:
i think we can just fmod like T angle_constrained = std::fmod(angle, 360); here

Reply:
lol, that's what I did initially, but 0 was sometimes 360, but I think I had an issue with my logging when I tested that.  I'll try it again.

Reply:
Okay, updated.